package.json

```diff
@@ -5,8 +5,8 @@
   "license": "MIT",
   "scripts": {
     "clean": "rm -rf node_modules packages/**/node_modules",
-    "build": "cd packages/contracts-bedrock && pnpm build",
-    "test": "cd packages/contracts-bedrock && pnpm test",
+    "build": "cd packages/contracts-bedrock && just build",
+    "test": "cd packages/contracts-bedrock && just test",
```

Comment:
Our root build and test scripts only run the ones in the contracts dir? Seems odd, I feel like it should run all test/build scripts. Do we use these root-level commands or should we just delete these two?

Reply:
These previously only built the `packages`, this keeps the same behavior. A follow up will fix this